rmgpy/molecule/converter.py

```diff
@@ -81,6 +83,14 @@ def to_rdkit_mol(mol, remove_h=True, return_mapping=False, sanitize=True):
         else:
             rd_atom_indices[atom] = index
 
+        # Check if a cutting label is present. If preserve this so that it is added to the SMILES string
+        if re.match(r'^[LR][0-9]*$', atom.label):  # Valid cutting labels include R, L, R#, L#, where # is any int
```

Comment:
I think you can replace this with an `isinstance(CuttingLabel)` check. This will keep us from having to update the code for re matching here as well

Reply:
Done!

Reply:
Done!